config/locales/en.yml

```diff
@@ -885,6 +886,45 @@ en:
       participate_in_campaign: Participate in a Campaign
       save_cloned_course: Save
       start_date: Start of activity tracking
+      scoping_methods:
+        scoping_methods: Scoping Methods
+        configure: Configure Scoping Methods
+        about: >
+          This is what lets the dashboard know which articles you wish to track. The articles tracked contribute to the course statistics and are displayed on the articles tab. Rather than adding each article you wish to track individually, you can add filters to track several of them at once.
+          
+          Say you only want to track pages created from the “Historic sites templates” or pages which belong to the category “Algebra” but not “Geometry”. With the correct scoping method, doing this is incredibly easy
+        categories: Categories
+        categories_short_desc: Categories are intended to group together pages on similar subjects. Categories are normally found at the bottom of an article page.
```

Comment:
"The Category feature on a wiki is used to collect related articles. You can use the name of the Category to tell the Dashboard to track all the articles in that Category."